Bio/PDB/parse_pdb_header.py

```diff
@@ -21,6 +20,23 @@
 from Bio import File
 
 
+# Added to parse SSBOND records from the PDB header and add them back to the header dict
+# Eric G. Suchanek, PhD 11/25/22
+def _get_ssbond(inl):
```

Comment:
This is a private function (leading underscore) so we don't insist on a docstring, but the comments here might as well be transformed into a docstring. Note there are style rules (one line summary, blank line, more text etc), so do run the pre-commit checks as per https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst